site/ru/README.md

```diff
@@ -1,25 +1,30 @@
-# Не Официальный Перевод
+# Неофициальный перевод
 
-Вся информация в этом разделе переведена при помощи русскоговорящего Tensorflow
-сообщества на общественных началах. По скольку этот перевод не является
+Вся информация в этом разделе переведена с помощью русскоговорящего Tensorflow
+сообщества на общественных началах. Поскольку этот перевод не является
 официальным, мы не гарантируем что он на 100% аккуратен и соответствует
 [официальной документации на английском языке](https://www.tensorflow.org/?hl=en).
-Если у вас есть предложение как исправить этот перевод, мы будем очень рады
+
+1. Если у вас есть предложение как исправить этот перевод, мы будем очень рады
 увидеть pull request в [tensorflow/docs](https://github.com/tensorflow/docs)
-GitHub репозиторий. Если вы хотите помочь сделать документацию по Tensorflow
-лучше (сделать сам перевод или проверить перевод подготовленный кем то другим),
+репозиторий GitHub. 
+
+2. Если вы хотите помочь сделать документацию по Tensorflow
+лучше (сделать сам перевод или проверить перевод подготовленный кем-то другим),
 заполните [эту форму](https://bit.ly/tf-translate) или напишите нам на
 [docs@tensorflow.org list](https://groups.google.com/a/tensorflow.org/forum/#!forum/docs).
 
 # Community translations
 
-Our TensorFlow community has translated these documents. Because community
+These docs were translated by our TensorFlow community. Because community
```

Comment:
Keep this as-is to use a more active voice.

Reply:
Got it, agree!